src/libcore/num/mod.rs

```diff
@@ -4523,24 +4517,34 @@ impl TryFromIntError {
     }
 }
 
-#[unstable(feature = "try_from", issue = "33417")]
+#[stable(feature = "try_from", since = "1.34.0")]
 impl fmt::Display for TryFromIntError {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
         self.__description().fmt(fmt)
     }
 }
 
-#[unstable(feature = "try_from", issue = "33417")]
+#[stable(feature = "try_from", since = "1.34.0")]
+impl From<Infallible> for TryFromIntError {
```

Comment:
@eddyb @arielb1 Can you double check that this impl isn't problematic re. coherence and adding `From<!> for T` later when `type Infallible = !;` (e.g. re. https://github.com/rust-lang/rust/issues/57012#issuecomment-460740678)? -- I think it should be OK but I'd like to be more certain.

Reply:
Since this impl is in the standard library, we get to remove it at the same time as whenever we add `From<!> for T`. It’s only compat with impls in other crates that we need to be worried about.